websockets/commands.go

```diff
@@ -300,3 +300,17 @@ type FeeResult struct {
 	MaxQueueSize uint32 `json:"max_queue_size,string"`
 	Status       string `json:"status"`
 }
+
+type ServerInfoCommand struct {
+	*Command
+	Result *ServerInfoResult `json:"result,omitempty"`
+}
+
+type ServerInfoResult struct {
+	Info ServerInfo `json:"info"`
+}
+
+type ServerInfo struct {
```

Comment:
Plenty more fields to add!

https://xrpl.org/server_info.html

Reply:
I was adding the basic information I was interested in only. Will keep this one private